src/propagation/parameters.jl

```diff
@@ -0,0 +1,96 @@
+struct Parameters{T <: AbstractFloat}
```

Comment:
I feel we can make the name `Parameters` a bit more descriptive, maybe `NEOParameters`, `NEOParameters`?

Reply:
Since the struct contains parameters used for many routines apart from propagation, I've decide to change all `Parameters` to `NEOParameters`.